### ISSUE TITLE
Github/Gitflow Tutorials

### ISSUE THREAD

**Issue by jonathan-freitas** [jonathan-freitas/ios-journey#4]
## [Video Tutorials from Ray Wender Lich - Beginning Git](https://www.raywenderlich.com/4418-beginning-git)
- [x] 1 - Introduction
- [x] 2 - Cloning a Repo
- [x] 3 - Creating a Repo
- [x] 4 - Creating a Remote
- [x] 5 - Commiting Changes
- [x] 6 - The Staging Area
- [x] 7 - Ignoring Files
- [x] 8 - Viewing History
- [x] 9 - Branching
- [x] 10 - Merging
- [x] 11 - Syncing with a Remote
- [x] 12 - Pull Requests
- [x] 13 - Conclusion

## [Video Tutorials from Ray Wender Lich - Mastering Git](https://www.raywenderlich.com/4289-mastering-git)
- [x] 1 - Implementation of Git
- [x] 2 - Merge Conflicts
- [x] 3 - Stashes
- [x] 4 - Aliases
- [x] 5 - Rebase: A Mega Alternative
- [x] 6 - Rebase: Rewriting History
- [x] 7 - Gitignore After the Fact
- [x] 8 - Cherry Picking
- [x] 9 - Filter Branch
- [x] 10 - Many Faces of Undo
- [x] 11 - GUIs: GITK
- [x] 12 - GUIs: Sourcetree
- [x] 13 - GUIs: GitUp
- [x] 14 - Conclusion

## [Atlassian Tutorial - Gitflow Workflow](https://br.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)
![screen shot 2018-11-09 at 15 14 12](https://user-images.githubusercontent.com/12562299/48277542-267ca200-e432-11e8-932f-94d5e08d0b67.png)

### The overall flow of Gitflow is:

1. A develop branch is created from master.
2. A release branch is created from develop.
3. Feature branches are created from develop.
4. When a feature is complete it is merged into the develop branch.
5. When the release branch is done it is merged into develop and master.
6. If an issue in master is detected a hotfix branch is created from master.
7. Once the hotfix is complete it is merged to both develop and master.

**Comment by jonathan-freitas** [comment]
GUIs done and commited today
